qlib/data/dataset/handler.py

```diff
@@ -685,7 +685,7 @@ def cast(cls, handler: "DataHandlerLP") -> "DataHandlerLP":
             the converted processed data
         """
         new_hd: DataHandlerLP = object.__new__(DataHandlerLP)
-        new_hd.from_cast = True  # add a mark for the casted instance
+        new_hd.from_cast = True  # add a mark for the cast instance
```

Comment:
Why not casted?

Reply:
The past tense of cast is: cast in past simple is cast. and past participle is cast. There is no `casted`.

https://engverbs.com/grammar/cast#:~:text=The%20past%20tense%20of%20cast%20is%3A%20cast%20in,cast.%20The%20past%20participle%20of%20cast%20is%20cast.

Reply:
Sorry for my bad grammar...